src/sagemaker/workflow/pipeline.py

```diff
@@ -125,6 +125,15 @@ def __init__(
             self.sagemaker_session.boto_session.client("scheduler"),
         )
 
+    @property
+    def latest_pipeline_version_id(self):
+        """Retrieves the latest version id of this pipeline"""
+        summaries = self.list_pipeline_versions(max_results=1)["PipelineVersionSummaries"]
+        if not summaries:
+            return None
+        else:
+            return summaries[0].get("PipelineVersionId")
```

Comment:
Is this always guaranteed to be the latest version when sort_order is None?

Reply:
Yes, the default sort order is descending